docs/rules/no-plusplus.md

```diff
@@ -32,16 +32,26 @@ j
 
 This rule is aimed at flagging the use of `++` and `--`. Some believe that the use of these unary operators reduces code quality and clarity. There are some programming languages that completely exclude these operators.
 
+### Options
+
+This rule, in it's default state, does not require any arguments. If you would like to enable one or more of the following you may pass an object with the options set as follows:
+
+* `forLoopAfterthoughts` set to `true` will allow you to use the unary operators `++` and `--` in the `afterthought` (or `final-expression`) of a `for loop`.
```

Comment:
I think the option probably needs to be named `allowForLoopAfterthoughts` to make it clear what it's doing.